WebClient/src/components/outsiderSuggestions/outsiderSuggestions.ts

```diff
@@ -329,6 +313,22 @@ export class OutsiderSuggestionsComponent {
         return [chor, phan, pony];
     }
 
+    private getBorbFant( ancientSouls: number, transcendentPower: number ): number {
+        transcendentPower = transcendentPower || (25 - 23 * Math.exp(-0.0003 * ancientSouls)) / 100;
```

Comment:
Is this line needed? Seems like `transcendentPower` would always be set at this point from line 87.